Project.toml

```diff
@@ -23,15 +23,15 @@ UnPack = "3a884ed6-31ef-47d7-9d2a-63182c4928ed"
 [compat]
 ArtifactWrappers = "0.2"
 ClimaComms = "0.3, 0.4, 0.5"
-ClimaCore = "0.10"
-ClimaCoreTempestRemap = "0.3.5, 0.3.6, 0.3.7"
+ClimaCore = "0.10.32"
```

Comment:
if this is reducing duplicates in compat entries in downstream folders, do we need to change the compat for the core Project.toml? Can you describe a little more how you got these versions?

Reply:
I tried compiling ClimaLSM using ClimaCore@0.10.0 (which would be allowed if we just had `ClimaCore = "0.10"`), but I got an error that ClimaCore.PointSpace wasn't defined [here](https://github.com/CliMA/ClimaLSM.jl/blob/main/src/ClimaLSM.jl#L9). I had a hard time finding exactly in which tag that function was added, but when I ran `add ClimaCore` with the oldest allowed versions of other packages, it added 0.10.32. I agree this isn't a very clear way to arrive at a version to specify, but it seems like we need something more recent than 0.10.0.